client/src/popups/SetDocumentToSavePoint.tsx

```diff
@@ -164,11 +143,11 @@ export function SetDocumentToSavePoint({
                 await doenetmlChangeCallback();
                 fetcher.submit(
                   {
-                    _action: "revert to revision",
+                    path: "updateContent/revertToRevision",
                     contentId,
                     revisionNum: revision.revisionNum,
                   },
-                  { method: "post" },
+                  { method: "post", encType: "application/json" },
```

Comment:
When click this button, the content is correctly reverted to the save point, but the modal stays at the "Use the save point" screen. I.e., the `setUpdate(true)` from line 52 is not being called.

Maybe the way it is triggering on a change in `fetcher.data` is brittle? It was probably specific to some way it was set up than is now changed.